NorthstarDLL/modmanager.cpp

```diff
@@ -161,6 +161,85 @@ Mod::Mod(fs::path modDir, char* jsonBuf)
 		}
 	}
 
+	if (modJson.HasMember("ConCommands") && modJson["ConCommands"].IsArray())
+	{
+		for (auto& concommandObj : modJson["ConCommands"].GetArray())
+		{
+			if (!concommandObj.IsObject() || !concommandObj.HasMember("Name") || !concommandObj.HasMember("Function") ||
```

Comment:
Would it be possible to have braces here? The fact the conditions span two lines make it very confusing.